db/migrate/20170330161442_change_institution_defaults.rb

```diff
@@ -0,0 +1,48 @@
+class ChangeInstitutionDefaults < ActiveRecord::Migration
+  def change
+    change_table :institutions do |t|
+      t.change :flight, :boolean, default: nil
+      t.change :correspondence, :boolean, default: nil
+      t.change :gibill, :integer, default: nil
+      t.change :yr, :boolean, default: nil
```

Comment:
For some boolean values that we search on, a `true` value is determined simply by the inclusion of the school in a data file. There is no guarantee that if it is not in the list, the boolean is false. The best we can say is that it isn't true.